src/index.js

```diff
@@ -43,8 +43,26 @@ function renderTodoApp() {
     h1.innerText = "Todo List";
 
     app.appendChild(h1);
-    // and maybe some here
+
     app.appendChild(ul);
+
+    let form = document.querySelector("#form");
+    let button = document.querySelector("#submit-button");
+
+    form.onsubmit = (e) => {
+        e.preventDefault();
+        let li = document.createElement("li");
+        let submission = document.querySelector("#form-input").value;
+        let input = `<input type="checkbox">${submission}</input>`;
```

Comment:
this method works (just making new html elements in the onclick for a new todo)
but we have already written a method that will create html for us given an array of `todos`. So ideally in this function, we would just add a new `todo` object to our current `todos` array and call our `renderTodoApp()` function again to update the page with our current todos list. This also makes it easier for us down the line if we wanted to do something like add the ability to remove or move a todo item; we can just manipulate the objects in our array, and call our render function, instead of having to mess with more html.